### PR TITLE
Fix demo page

### PR DESCRIPTION
**Pull request by mathsalmi**
Currently the demo page doesn't work. On the console it shows the following:

```
SyntaxError: Can't create duplicate variable that shadows a global property: 'language'
```

This PR fixes this problem.

### CODE REVIEW

**Comment by ukupat** [comment]
Vau, thank you for your contribution! I didn't know anything about this error. What browser and OS are you using?

I can't merge this PR because `index.html` script is not called (wrapper function is declared but never called) thus demo is not working. Can you fix it, @mathsalmi?

File: index.html [hunk 1]
```diff
@@ -108,21 +108,23 @@
 
     <script type="text/javascript" src="https://unpkg.com/sql-formatter@1.1.2/dist/sql-formatter.min.js"></script>
     <script>
-        let language = document.getElementById('language');
-        let input = document.getElementById('input');
-        let output = document.getElementById('output');
+        (function() {
```
Comment (ukupat) [inline index.html]: Wrapper function is not called.

**Comment by mathsalmi** [comment]
Oh what a genius I am. I'll fix it right away! 

Btw, I'm using Safari (10.0.3) on macOS Sierra (10.12.3)

**Comment by ukupat** [comment]
Aaa, Safari, now I understand. Thanks again for your help, @mathsalmi!

**Pull request merged**